htdocs/lib2/logic/geodb.class.php

```diff
@@ -26,8 +26,7 @@ public static function setAllCacheLocations()
 
         sqll("
             DELETE FROM `cache_location`
-            WHERE `cache_id` NOT IN (SELECT `cache_id` FROM `caches`)"
-        );
+            WHERE `cache_id` NOT IN (SELECT `cache_id` FROM `caches`)");
     }
```

Comment:
One PSR-2 violation (beginning " on the wrong line) has been converted into two (closing brace also on the wrong line).